build/cmake/modules/FindLIBSOUP.cmake

```
@@ -31,19 +31,24 @@
 # LibSoup does not provide an easy way to retrieve its version other than its
 # .pc file, so we need to rely on PC_LIBSOUP_VERSION and REQUIRE the .pc file
 # to be found.
+SET(LIBSOUP_VERSION 2.4)
+if(DEFINED LIBSOUP_FIND_VERSION)
+    SET(LIBSOUP_VERSION ${LIBSOUP_FIND_VERSION})
+endif()
+
```

I tried to update an existing CMake project from webkitgtk-4.0/libsoup-2.4 to webkitgtk-4.1/libsoup-3.0. It correctly found the packages, but did not update the cached variables. Could you add these lines so the variables will be updated? 
FindWEBKIT2 already has this.

```suggestion

set(LIBSOUP_INCLUDE_DIRS LIBSOUP_INCLUDE_DIRS-NOTFOUND)
set(LIBSOUP_LIBRARIES LIBSOUP_LIBRARIES-NOTFOUND)

```

---

Thanks for the comment.  Incorporated.